File: dayFour/afterBreak/java/TaskNine.java
import java.util.*;


public class TaskNine {
	static Scanner input = new Scanner(System.in);
	
	public static void main() {
	
	int validSum = 0;
	
	System.out.println("Enter the scores below: ");
	
	
	for(int counter= 0; counter < 10; counter++)
	  {
	    int score = input.nextInt();
	    if(score >= 0 && score <= 100)
	      {
	        validSum += score;
	      } else { System.out.println("Invalid score");}
	  }
	  
	System.out.println("The sum of valid scores is: " + validSum);
	}
}

File: dayFour/afterBreak/python/taskNine.py
sum_total = 0

print("Enter your scores below: ")
for scores in range(1, 11):
    score = int(input())
    if score >= 0 and score <= 100:
        sum_total = sum_total + score
    else:
        print('Invalid Score')
print("The sum of scores is:", sum_total)
